app/components/header.hbs

```diff
@@ -0,0 +1,69 @@
+<section class='header'>
+  <img
+    class='header__logo'
+    src='assets/images/rds-logo-2x.png'
+    alt='Real Dev Squad'
+  />
+
+  <div class='header__container'>
+    <span class='container__text-social'>
+      <h2 class='text__subtitle'>
+        <span class='text__welcome'>Welcome</span>
+        <span class='text__to'>to the</span>
+      </h2>
+      <h1 class='text__title'>Real Dev Squad</h1>
+
+      <div class='social'>
+        <a href={{this.LINKEDIN_URL}} target='_blank' class='social__handles'>
+          LinkedIn
+          <img
+            class='social__icon'
+            src='assets/icons/linkedIn-logo.png'
+            alt='LinkedIn Icon'
+          />
+        </a>
+
+        <span class='social__seperators-vertical'></span>
+        <span class='social__seperators-round'>&bull;</span>
+
+        <a href={{this.TWITTER_URL}} target='_blank' class='social__handles'>
+          Twitter
+          <img
+            class='social__icon'
+            src='assets/icons/twitter-logo.png'
+            alt='Twitter Icon'
+          />
+        </a>
+
+        <span class='social__seperators-vertical'></span>
+        <span class='social__seperators-round'>&bull;</span>
+
+        <a href={{this.FACEBOOK_URL}} target='_blank' class='social__handles'>
+          Facebook
+          <img
+            class='social__icon'
+            src='assets/icons/facebook-logo.png'
+            alt='Facebook Icon'
+          />
+        </a>
```

Comment:
Could we follow DRY(Don't Repeat Yourself) here?
Could we not look into having constant socialLinkProperties where we could have properties like href link, name of the link, image src and alt tag.
We could have one source of data there and could pass that in a function here

Reply:
Yes, made the following change